### DISCUSSION TITLE
Running minimize with Lammps will repeatedly calculate the model deviation

### DISCUSSION THREAD

**Discussion by jpqiu** [deepmodeling/deepmd-kit#3590] [Q&A]
When I run the minimize operation in Lammps, the log.lammps file outputs normally, but I've noticed that the model_devi.out file exhibits repetitive calculations. I'm trying to understand the cause of this issue. For instance, while log.lammps indicates that 567 steps have been executed, the model_devi.out file contains 1129 lines.Furthermore, the model deviation calculated for the same number of steps is not consistent.
I am using version 2.2.7. Below are my input settings along with the log.lammps and model_devi.out files.

<img width="1317" alt="image" src="https://github.com/deepmodeling/deepmd-kit/assets/148757424/7b024e58-67cc-4abf-80c2-f09052d0c22b">

[input_output.zip](https://github.com/deepmodeling/deepmd-kit/files/14732061/input_output.zip)

**Comment by jpqiu** [comment]
It's possible that the CG algorithm also outputs the model deviation of internal calculations, which is not conducive to my sampling. Is there any way to correct this?

**Reply by njzjz** [reply]
We rejected the idea to distinguish the internal calculations in #1311 due to side effects.

**Reply by jpqiu** [reply]
Thank you for your reply
